components/layout_thread/lib.rs

```diff
@@ -689,6 +695,10 @@ impl LayoutThread {
             Msg::SetFinalUrl(final_url) => {
                 self.url = final_url;
             },
+            Msg::SetPaintWorkletExecutor(executor) => {
+                debug!("Setting the paint worklet executor");
+                self.paint_worklet_executor = Some(executor);
```

Comment:
`debug_assert!(self.paint_worklet_executor.is_none())`? This would need to be refactored if/when we fix https://github.com/servo/servo/issues/14885